modules/codemod/lib/v6/spec/deprecateCookieBanner.spec.ts

```diff
@@ -0,0 +1,108 @@
+import {expectTransformFactory} from './expectTransformFactory';
+import transformer from '../deprecateCookieBanner';
+const context = describe;
+
+const expectTransform = expectTransformFactory(transformer);
+
+describe('Canvas Kit Deprecate CookieBanner Codemod', () => {
+  context('when transforming CookieBanner imports', () => {
+    it('should ignore non-canvas-kit imports', () => {
+      const input = `import {CookieBanner} from '@workday/some-other-library'`;
+      const expected = `import {CookieBanner} from '@workday/some-other-library'`;
```

Comment:
I realize this is probably consistent with other "non-change" specs but something we could do differently is just reference the `input`:
`const expected = input;`

Reply:
```suggestion
      const expected = input;
```